src/component/feature/FeatureView/FeatureMetrics2/FeatureMetricsHours.tsx

```diff
@@ -0,0 +1,43 @@
+import GeneralSelect, {
+    OnGeneralSelectChange,
+} from '../../../common/GeneralSelect/GeneralSelect';
+
+export type FeatureMetricsHour = 1 | 24 | 48;
+
+export const FeatureMetricsHours = (props: {
+    hoursBack: number;
+    setHoursBack: (value: FeatureMetricsHour) => void;
+}) => {
+    const onChange: OnGeneralSelectChange = event => {
+        props.setHoursBack(parseFeatureMetricsHour(event.target.value));
+    };
+
+    return (
+        <GeneralSelect
+            label="Period"
+            name="feature-metrics-period"
+            id="feature-metrics-period"
+            options={hourOptions}
+            value={String(props.hoursBack)}
+            onChange={onChange}
+            fullWidth
+        />
+    );
+};
+
+const parseFeatureMetricsHour = (value: unknown): FeatureMetricsHour => {
```

Comment:
It would be good if we folderized the subcomponents so that it's clear which component is the main component here and consumes the others. This has the added benefit of being able to extract helper functions into separate files that you can import into the component you are using without creating too much clutter. As well as being able to easily delete parts of the component if we need to refactor something. 

Alternatively, if it's just a small helper function. Moving functions and variables to the top of the component will be in line with previous components.